src/index.html

```diff
@@ -5,8 +5,449 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="http://localhost:8080"
+        class="header__icon"
+      >
+        <img
+          src="./images/logo.png" 
+          alt="header__icon"
+        >
```

Comment:
you should move each attribute on new line in 2 cases:
1. you have 3 and more attributes
2. your line is more than maximum allowable length of line